### DISCUSSION TITLE
Dashboard Graphs not showing max Values

### DISCUSSION THREAD

**Discussion by NZ-Walker** [datalust/seq-tickets#2227] [Help]
Hi All, 

I keep running into situations where the Y-Axis scale on my graphs don't show the maximum values for a given time period (image below). If I adjust the time period to that specific area it will re-scale and show the max value correctly.

Is there any way to force the Y-Axis to scale in such a way that it will always show the max values across all time periods instead of the default behavior which appears to be based on an average across all values in the given time period?

<img width="597" alt="Seq Graph Max Values" src="https://github.com/datalust/seq-tickets/assets/59713778/48c18507-c392-4f0d-8bf9-b884ac552fc4">

**Comment by KodrAus** [comment]
Thanks for the report @NZ-Walker. This isn't something I've seen before. It definitely doesn't look right. You shouldn't have to make any special configuration to your chart for it to show you all the data on it; it should really just do it for you.

I've converted this into a regular bug report in #2228

**Comment by KodrAus** [comment]
Also worth checking just in case, are you running on the current (2024.3) version of Seq @NZ-Walker?

**Reply by NZ-Walker** [reply]
Of course I should have checked this and didn't... the issue doesn't occur on 2024.3 the screen shot above was on 2024.2

Thanks @KodrAus
